OpenSim/Common/Set.h

```diff
@@ -100,22 +93,21 @@ Set() :
  * @param aFileName             Name of the file.
  * @param aUpdateFromXMLNode    Whether to update from XML.
  */
-Set(const std::string &aFileName, bool aUpdateFromXMLNode = true) :
-    Object(aFileName),
+Set(const std::string &aFileName, bool aUpdateFromXMLNode = true) : Super(aFileName),
```

Comment:
We need to annotate this with SWIG_DECLARE_EXCEPTION so that file loading failures are caught in proper context.

Reply:
Good catch! Please feel free to add it. One good thing is that it is now centralized instead distributed amongst the concrete objects. Could this not be pushed even further down to Object itself? We could modify the exception to:
1. be more specific (e.g. FileNotFound), and 
2. include the concrete class name so the message is more meaningful.

Reply:
I'll add this now.